### PR TITLE
Fix up recent ng upgrade

### PR DESCRIPTION
**Pull request by jimmidyson**
Have to run through `ng update` on upgrade and resolve conflicts... fun!

Now linting should work again in CI so be prepared for annoying breaking builds - you'll get used to it though!

### CODE REVIEW

**Comment by kahboom** [comment]
@jimmidyson - No need to get used to it, I run it before every PR when I can :)

File: .gitignore [hunk 1]
```diff
@@ -6,7 +6,6 @@
 
 # dependencies
 /node_modules
-/bower_components
```
Comment (kahboom) [inline .gitignore]: How in the..

File: angular-cli.json [hunk 2]
```diff
@@ -53,6 +54,16 @@
       "config": "./protractor.conf.js"
     }
   },
+  "lint": [
```
Comment (kahboom) [inline angular-cli.json]: 👍

File: src/app/integrations/create-page/configure-connection/configure-connection.component.ts [hunk 3]
```diff
@@ -2,10 +2,10 @@ import { Component, OnInit } from '@angular/core';
...
-  templateUrl: 'configure-connection.component.html'
+  templateUrl: 'configure-connection.component.html',
 })
 export class IntegrationsConfigureConnectionComponent implements OnInit {
   constructor() { }
 
   ngOnInit() { }
-}
\ No newline at end of file
+}
```
Comment (kahboom) [inline src/app/integrations/create-page/configure-connection/configure-connection.component.ts]: Thanks, this one drives me nuts.

**Comment by jimmidyson** [comment]
@kahboom It's not you I was worried about :-b

File: src/app/integrations/create-page/create-page.component.ts [hunk 4]
```diff
@@ -27,10 +27,10 @@ export class IntegrationsCreatePage implements OnInit, OnDestroy {
 
   ngOnInit() {
     this.subscription = this.route.params.pluck<Params, string>('integrationId')
-      .map((integrationId:string) => this.store.loadOrCreate(integrationId))
+      .map((integrationId: string) => this.store.loadOrCreate(integrationId))
       .subscribe();
-    this.integration.subscribe((i:Integration) => {
-      console.log("Integration: ", i);
+    this.integration.subscribe((i: Integration) => {
+      console.log('Integration: ', i);
```
Comment (kahboom) [inline src/app/integrations/create-page/create-page.component.ts]: Is it my turn to pick on you for `console.log()`'s now? ;D

Comment (jimmidyson) [inline src/app/integrations/create-page/create-page.component.ts]: Ha - we should lint for that. Can we? We should be able to. Let me check.

**Pull request merged**